Jenkinsfile_nightly

```diff
@@ -1,15 +1,22 @@
 #!groovy
+
 properties([
-    pipelineTriggers([]), #! Paused add [cron('15 5 * * *')] to make have the build working again.
-    parameters([
-        string(name: 'URL_TO_TEST', defaultValue: 'https://immigration-appeal.aat.platform.hmcts.net', description: 'The URL you want to run these tests against'),
+    [
+        $class       : 'GithubProjectProperty',
+        displayName  : 'AIP Frontend',
+        projectUrlStr: 'https://github.com/hmcts/ia-aip-frontend/'
+    ],
+    pipelineTriggers([
+    cron('15 5 * * *')
     ])
 ])
 
 @Library("Infrastructure")
 
-def product = "ia"
+def type      = "nodejs"
+def product   = "ia"
 def component = "aip-frontend"
+def channel   = "#aip-tech"
```

Comment:
let's use `ia-tech` channel, not sure if aip one is created

Reply:
you are correct, this channel does not (or no longer) exist. have corrected and pushed